### ISSUE TITLE
Credential Banner modal is not dismissed after saving

### ISSUE THREAD

**Issue by stdedos** [keepassxreboot/keepassxc-browser#2320]
## Expected Behavior

Finishing a final action (New / Update / Cancel), the Credential Banner modal does not disappear.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Debug info

KeePassXC - 2.7.9
KeePassXC-Browser - 1.9.2
Operating system: Linux x86_64
Browser: Chrome/Chromium 114.0.0.0

```
 [Debug observer-helper.js:195] KeePassXC-Browser - Input fields found:
09:49:01.106  Array(3)
09:49:01.106  [Debug keepassxc-browser.js:344] KeePassXC-Browser - Login field combinations identified:
09:49:01.106  Array(2)
09:49:01.107  [Debug observer-helper.js:195] KeePassXC-Browser - Input fields found:
09:49:01.107  Array(3)
09:49:01.107  [Debug keepassxc-browser.js:344] KeePassXC-Browser - Login field combinations identified:
09:49:01.107  Array(2)
09:49:01.207  [Debug observer-helper.js:195] KeePassXC-Browser - Input fields found:
09:49:01.207  Array(3)
09:49:01.207  [Debug keepassxc-browser.js:344] KeePassXC-Browser - Login field combinations identified:
09:49:01.207  Array(2)
09:49:01.208  [Debug observer-helper.js:195] KeePassXC-Browser - Input fields found:
09:49:01.208  Array(3)
09:49:01.208  [Debug keepassxc-browser.js:344] KeePassXC-Browser - Login field combinations identified:
09:49:01.208  Array(2)
09:49:01.216  [Debug fill.js:30] KeePassXC-Browser - Error: Credential list is empty for: https://www.flightchecklists.com
09:49:01.216  [Debug ui.js:297] KeePassXC-Browser - No logins found. https://www.flightchecklists.com
09:49:08.062  [Debug observer-helper.js:195] KeePassXC-Browser - Input fields found:
09:49:08.063  Array(1)0: input#mui-13.MuiInputBase-input.MuiOutlinedInput-input.MuiInputBase-inputSizeSmall.css-1o6z5nglength: 1[[Prototype]]: Array(0)
09:49:08.103  [Debug keepassxc-browser.js:500] KeePassXC-Browser - Error: No combination found.
09:49:10.072  [Debug ui.js:297] KeePassXC-Browser - New credentials with username vella_larson@Kurzepost.de saved.
09:49:12.604  [Debug ui.js:297] KeePassXC-Browser - New credentials with username vella_larson@Kurzepost.de saved.
09:49:12.984 chrome-extension://o…ontent/banner.js:29 Uncaught (in promise) TypeError: Failed to execute 'removeChild' on 'Node': parameter 1 is not of type 'Node'.
    at kpxcBanner.destroy (chrome-extension://oboonakemofpalcgghocfoadofidjkkk/content/banner.js:29:37)
09:49:16.900  [Debug ui.js:297] KeePassXC-Browser - New credentials with username (empty) saved.
09:49:18.875  [Debug ui.js:297] KeePassXC-Browser - New credentials with username (empty) saved.
10:03:54.950  [Debug observer-helper.js:195] KeePassXC-Browser - Input fields found:
10:03:54.950  Array(1)
10:03:54.950  [Debug observer-helper.js:195] KeePassXC-Browser - Input fields found:
10:03:54.950  Array(1)
```

**Comment by varjolintu** [comment]
I cannot reproduce this.

**Comment by droidmonkey** [comment]
I can't either, it works great

**Comment by stdedos** [comment]
I am not sure what other diagnosing information to provide either 🤷 

The extension ended up creating 4 similar entries (all with wrong password, since the first attempt was not accepted by the complexity check and I had to amend)

**Comment by varjolintu** [comment]
So this happens only after a new credentials are created? Not if you just submit any credentials to the login fields and try to login etc?

**Comment by stdedos** [comment]
> So this happens only after a new credentials are created?

I was quite frugal with my description 😓 

1. Yes, I tried to create new credentials to the website.
2. "Save it?" modal appears (for a lack of a better word)
3. I tried to create New credentials (evidently, from the logs, there are none)
4. No amount of button-mashing would make it go away (or save the "correct" password)

-----

Seems to happen also when having to update the password (via the extension's Update modal).
(Remember from https://github.com/keepassxreboot/keepassxc-browser/issues/2320#issuecomment-2308782906 that the wrong password is stored at the DB)
